src/Console/Commands/ShortenCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class ShortenCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'shorten';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Shorten a given URL';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $driver = $this->laravel['url.shortener']->driver($this->input->getOption('driver'));
```

Comment:
Use dependency injection here

Reply:
Between Laravel 5.5 and 5.6 the place of the dependency injection changed.

https://laravel.com/docs/5.5/artisan#command-structure

> Note that we are able to inject any dependencies we need into the command's constructor.

https://laravel.com/docs/5.6/artisan#command-structure

> Note that we are able to inject any dependencies we need into the command's constructor or `handle` method.

If you want to keep compatibility you have to use this way.

Reply:
Not necessarily, seems that all versions allow injection via the controller